### PR TITLE
JENA-1960: Rewrite Fuseki examples for new style endpoints

### PR DESCRIPTION
**Pull request by afs**
... and link jena-fuseki-docker assembly into the build.

### CODE REVIEW

File: jena-fuseki2/examples/config-1-mem.ttl [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    fuseki:endpoint [ 
+        fuseki:operation fuseki:upload ;
+        fuseki:name "upload"
+    ] ; 
+    fuseki:dataset :dataset ;
+    .
+
+# Transactional in-memory dataset.
+:dataset rdf:type ja:MemoryDataset ;
+	## Optional load with data on start-up
```
Comment (kinow) [inline jena-fuseki2/examples/config-1-mem.ttl:50]: tab for space

File: jena-fuseki2/examples/config-2-mem-old.ttl [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    fuseki:serviceUpdate               "update";
+    fuseki:serviceUpload               "upload" ;
+    fuseki:serviceReadWriteGraphStore  "data" ;
+    fuseki:serviceReadGraphStore       "get" ;
+    fuseki:dataset :dataset ;
+    .
+
+# Transactional in-memory dataset.
+:dataset rdf:type ja:MemoryDataset ;
+	## Optional load with data on start-up
```
Comment (kinow) [inline jena-fuseki2/examples/config-2-mem-old.ttl:30]: tab for space

File: jena-fuseki2/examples/config-timeout-server.ttl [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+PREFIX fuseki:  <http://jena.apache.org/fuseki#>
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ja:      <http://jena.hpl.hp.com/2005/11/Assembler#>
+PREFIX tdb2:    <http://jena.apache.org/2016/tdb#>
+
+## Server-wide timeout (all datasets).
+
+[] rdf:type fuseki:Server ;
+    # Format 1: "10000" -- 1 second timeout
```
Comment (kinow) [inline jena-fuseki2/examples/config-timeout-server.ttl:13]: 1000 ms? or 10 seconds timeout?

File: jena-fuseki2/examples/config-inference-2.ttl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ja:      <http://jena.hpl.hp.com/2005/11/Assembler#>
+PREFIX tdb2:    <http://jena.apache.org/2016/tdb#>
+
+[] rdf:type fuseki:Server ;
+   fuseki:services (
+     :service
+   ) .
+
+# Service description for "/dataset" with all endpints.
```
Comment (kinow) [inline jena-fuseki2/examples/config-inference-2.ttl:18]: s/endpints/endpoints :beer:

File: jena-fuseki2/examples/config-inference-1.ttl [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+PREFIX rdf:     <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs:    <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ja:      <http://jena.hpl.hp.com/2005/11/Assembler#>
+
+[] rdf:type fuseki:Server ;
+   fuseki:services (
+     :service
+   ) .
+
+# Service description for "/dataset" with all endpints.
```
Comment (kinow) [inline jena-fuseki2/examples/config-inference-1.ttl:18]: s/endpints/endpoints

File: jena-fuseki2/examples/config-inference-2.ttl [hunk 6]
```diff
@@ -0,0 +1,52 @@
+## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
+
+# Example of a data service with SPARQL query abnd update on an
```
Comment (kinow) [inline jena-fuseki2/examples/config-inference-2.ttl:3]: s/abnd/and ?

**Review by kinow — APPROVED** [review]

File: jena-fuseki2/examples/config-inference-2.ttl (hunk 4 above)
Comment (afs) [reply jena-fuseki2/examples/config-inference-2.ttl]: :beer: :beer:

**Pull request merged**